Doc/library/csv.rst

```diff
@@ -401,8 +401,10 @@ Reader objects (:class:`DictReader` instances and objects returned by the
 
 .. method:: csvreader.__next__()
 
-   Return the next row of the reader's iterable object as a list, parsed according
-   to the current dialect.  Usually you should call this as ``next(reader)``.
+   Return the next row of the reader's iterable object as a list (if the object
+   was returned from ``reader()``) or a dict (if it is a ``DictReader``), parsed
```

Comment:
`reader()` is not very clear here. "[...] from :func:\`reader\`" would be much clearer.